imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -110,37 +110,17 @@ class HistogramL0(GlowsL0):
     ELAVG: int
     ELVAR: int
     EVENTS: int
-    HISTOGRAM_DATA: bytes
-
-    def __init__(self, packet, software_version: str, packet_file_name: str):
-        """Initialize data class with a packet of histogram data.
-
-        Parameters
-        ----------
-        packet
-            Packet generated from space_packet_parser. Should be a type NamedTuple
-            with header and data fields.
-        software_version: str
-            The version of the ground software being used
-        packet_file_name: str
-            The filename of the packet
-        """
-        super().__init__(software_version, packet_file_name, CcsdsData(packet.header))
-
-        attributes = [field.name for field in fields(self)]
-
-        # For each item in packet, assign it to the matching attribute in the class.
-        for key, item in packet.data.items():
-            value = (
-                item.derived_value if item.derived_value is not None else item.raw_value
-            )
-            if key in attributes:
-                setattr(self, key, value)
-            else:
-                raise KeyError(
-                    f"Did not find matching attribute in Histogram data class for "
-                    f"{key}"
+    HISTOGRAM_DATA: bytearray
+
+    def __post_init__(self):
+        """Convert HISTOGRAM_DATA attribute from string to bytearray if needed."""
+        if isinstance(self.HISTOGRAM_DATA, str):
+            # Convert string output from space_packet_parser to bytearray
+            self.HISTOGRAM_DATA = bytearray(
```

Comment:
That's a good point, I left it as a bytearray because this class is supposed to be the raw Level 0 data, so it's the closest type representation to the decommed data. It gets converted into a list in L1A (although it probably should be a numpy array there)